clients/privacy-center/common/property-id.ts

```diff
@@ -0,0 +1,43 @@
+import { UserGeolocation } from "fides-js";
+import { NextApiRequest } from "next";
+import { PrivacyCenterEnvironment } from "~/app/server-environment";
+
+/**
+ * Verifies that certain conditions are met to be able to fetch experiences by property ID
+ */
+export const verifyConditionsForPropertyId = (
+  req: NextApiRequest,
+  geolocation: UserGeolocation | null,
+  environment: PrivacyCenterEnvironment,
+  fidesString: string | null
+) => {
+  const propertyId = req.query.property_id;
+
+  if (propertyId !== undefined) {
```

Comment:
Super-nit, very stylistic based but-- in general I'd recommend testing for what you *support* and throw errors if it doesn't match. That's a slightly more defensive approach.

Eg. instead of testing that it's not undefined and not an Array, instead test that it is *exactly* a string